apparmor.d/groups/systemd/systemd-logind

```diff
@@ -105,5 +108,56 @@ profile systemd-logind @{exec_path} flags=(attach_disconnected,complain) {
   /dev/shm/{,**/} rw,
   /dev/tty[0-9]* rw,
 
+  # DBus
```

Comment:
I'm trying not to give logind access to whole DBus. Afraid it's just an introduction.